src/current/v24.1/physical-cluster-replication-overview.md

```diff
@@ -1,62 +1,69 @@
 - **Monitoring**: To monitor the replication's initial progress, current status, and performance, you can use metrics available in the [DB Console]({% link {{ page.version.version }}/ui-overview.md %}) and [Prometheus]({% link {{ page.version.version }}/monitor-cockroachdb-with-prometheus.md %}). For more detail, refer to [Physical Cluster Replication Monitoring]({% link {{ page.version.version }}/physical-cluster-replication-monitoring.md %}).
-- **Data verification on standby**: You can verify that the data on the standby cluster matches that on the primary by checking the fingerprint of the data on each cluster. We recommend running data verification checks regularly as part of your monitoring processes. Refer to [Data verification]({% link {{ page.version.version }}/physical-cluster-replication-monitoring.md %}#data-verification) page for a guide and considerations.
 
 {{site.data.alerts.callout_info}}
-[Cutting over to a timestamp in the past]({% link {{ page.version.version }}/cutover-replication.md %}#cut-over-to-a-point-in-time) involves reverting data on the standby cluster. As a result, this type of cutover takes longer to complete than cutover to the [latest replicated time]({% link {{ page.version.version }}/cutover-replication.md %}#cut-over-to-the-most-recent-replicated-time). The increase in cutover time will correlate to how much data you are reverting from the standby. For more detail, refer to the [Technical Overview]({% link {{ page.version.version }}/physical-cluster-replication-technical-overview.md %}) page for physical cluster replication.
+[Cutting over to a timestamp in the past]({% link {{ page.version.version }}/cutover-replication.md %}#cut-over-to-a-point-in-time) involves reverting data on the standby cluster. As a result, this type of cutover takes longer to complete than cutover to the [latest replicated time]({% link {{ page.version.version }}/cutover-replication.md %}#cut-over-to-the-most-recent-replicated-time). The increase in cutover time will correlate to how much data you are reverting from the standby. For more detail, refer to the [Technical Overview]({% link {{ page.version.version }}/physical-cluster-replication-technical-overview.md %}) page for PCR.
 {{site.data.alerts.end}}
 
 ## Known limitations
 
 {% include {{ page.version.version }}/known-limitations/physical-cluster-replication.md %}
 
+## Performance
+
+Cockroach Labs supports PCR up to the following scale:
+
+- Cluster size: 30TB
+- Nodes: 7
+- Transaction payload: [TPC-C]({% link {{ page.version.version }}/cockroach-workload.md %}#tpcc-workload)
```

Comment:
@alicia-l2 Should we break this list up into "Cockroach Labs Supports: ..." which just lists cluster size metrics (TB, nodes, WPS) and then "Our testing with the TPC-C has demonstrated that:..." ? Because we _support_ far more than just the tpcc workload; we support any sql workload you want, as long as it doesn't include large schema changes or imports. We just only verified performance with the tpcc workload.

Reply:
Yeah agreed, except I think we should take nodes out and keep RPS.